src/index.html

```diff
@@ -6,9 +6,61 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="index.html" class="header__homelink">
+          <img
+           src="./images/logo.png"
+           class="logo"
+           alt="logo"
+           >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item" >
+            <a class="nav__link"
+            data-qa ="hover"
+            href="#laptops&computers"
+            >
+            Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              class="nav__link"
              data-qa ="hover"
              href="#laptops&computers"
            >
              Laptops & Computers
            </a>
```

Reply:
<img width="1019" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/27afaf79-c717-4112-a83c-16dc5dac35f4">

Reply:
Fix in accordance with the suggestion.